### PR TITLE
fix: remove aync-graphql fork

### LINKED ISSUE #1501: update async_graphql
async_graphql is pointing to a fork which it doesn't need to. The additional features have been merged and released.

### PR DESCRIPTION
**Pull request by neo773**
**Issue Reference(s):**  
Fixes #1501

**Build & Testing:**

- [ ] I ran `cargo test` successfully.
- [ ] I have run `./lint.sh --mode=fix` to fix all linting issues raised by `./lint.sh --mode=check`.

/claim #1501

### CODE REVIEW

File: Cargo.toml
Comment (ssddOnTop) [inline Cargo.toml:1]: I think you need to change it for async-graphql-value as well

Comment (ssddOnTop) [reply Cargo.toml:1]: @neo773 change it for cloudflare as well pls

File: Cargo.toml [hunk 1]
```diff
@@ -190,11 +176,6 @@ debug = false
 incremental = false
 overflow-checks = false
 
-[profile.release.package.cloudflare]
-strip = true
-codegen-units = 1
-opt-level = 'z'
-
```
Comment (tusharmath) [inline Cargo.toml:197]: these changes needed?

Comment (neo773) [reply Cargo.toml:197]: nope, not sure how it committed those

Comment (tusharmath) [reply Cargo.toml:197]: Can you revert them please.

**Pull request closed without merging**